File: backend/backend/Models/Locker.cs
using backend.Models;

public class Locker
{
    public Guid Id { get; set; }
    public string LocationCode { get; set; }
    public double Height { get; set; }
    public double Width { get; set; }
    public double Length { get; set; }
    public LockerState LockerState { get; set; }

    public Guid ParcelLockerId { get; set; }
    public ParcelLocker ParcelLocker { get; set; } = null!;

    public ICollection<IssueCompartment> IssueCompartments { get; set; } = new List<IssueCompartment>();

    // TODO: Implement Issue compartment 
    // public Guid IssueCompartmentId { get; set; }
    // public IssueCompartment IssueCompartment { get; set; }
}

File: frontend/src/entities/Locker.ts
import { Guid } from "guid-typescript";
import type { LockerState } from "./LockerState";

export type Locker = {
  id: Guid,
  locationCode: string,
  height: number, 
  width: number,
  length: number,
  lockerState: LockerState,
  parcelLockerId: number
};

// TODO: Naudoti Šitus DTO kaip backe o ne form content
// export type LockerCreateDto = Omit<Locker, "id" | "parcelLockerId" | "lockerState">
export type LockerUpdateDto = Omit<Locker, "id" | "parcelLockerId">